### PR TITLE
feat: support laravel-enum v2

### PR DESCRIPTION
**Pull request by atymic**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,30 +2,35 @@
 
 All notable changes to `nova-enum-field` will be documented in this file.
 
-## 1.0.0 - 2019-09-07
+## 2.0.0 - 2020-07-08
```
Comment (atymic) [inline CHANGELOG.md:5]: I reordered this to be newest-oldest. Saves having to scroll down to see the lastest release and in line with most other changelogs.

Comment (mdpoulter) [reply CHANGELOG.md:5]: This is great, thanks!

File: README.md [hunk 2]
```diff
@@ -22,21 +22,14 @@ composer require simplesquid/nova-enum-field
...
 {
-    use CastsEnums;
-
-    protected $enumCasts = [
-        // 'attribute_name' => Enum::class
-        'user_type' => UserType::class,
-    ];
-
     protected $casts = [
         'user_type' => 'int',
```
Comment (mdpoulter) [inline README.md:31]:
```suggestion
```

I think this cast can now be removed too.

File: composer.json [hunk 3]
```diff
@@ -28,12 +28,12 @@
     "require": {
         "php": "^7.3",
         "laravel/nova": "^2.0|^3.0",
```
Comment (mdpoulter) [inline composer.json:30]:
```suggestion
        "laravel/nova": "^3.0",
```

Is Nova 2 supported by Laravel 7? If not, we might as well drop support for it too.

File: README.md (hunk 2 above)
Comment (atymic) [reply README.md:31]: You're right, done :)

File: composer.json (hunk 3 above)
Comment (atymic) [reply composer.json:30]: Also correct, have dropped it :)

**Pull request merged**